lib/elixir/lib/enum.ex

```diff
@@ -461,6 +461,26 @@ defmodule Enum do
     end) |> elem(1)
   end
 
+
+  @doc """
+  Enumerates the collection, removing adjacent duplicated elements.
```

Comment:
This sounds a bit ambiguous, as if we were removing all duplicated elements instead of leaving only one copy.

Some other options:

"Returns a list with all runs of equal elements in the original collection compressed to/replaced by a single element."

"Enumerates the collection, deduplicating adjacent elements. That is, if there are two or more equal elements next to each other, only one such element will be left in the result."